scripts-dev/release.py

```diff
@@ -69,11 +69,12 @@ def cli():
         # ... wait for assets to build ...
 
         ./scripts-dev/release.py publish
+
```

Comment:
This newline is needed, annoyingly, as otherwise it renders the two commands on a single line

Reply:
Oh fun, TIL.